auto_nag/bugbug_utils.py

```diff
@@ -16,6 +16,40 @@
 )
 
 
+def get_bug_ids_classification(
+    model, bug_ids, bugs=None, retry_count=100, retry_sleep=1
+):
+    if len(bug_ids) > 0:
+        url = f"{BUGBUG_HTTP_SERVER}/{model}/predict/batch"
+
+        for i in range(retry_count):
+            response = requests.post(
+                url, headers={"X-Api-Key": "Test"}, json={"bugs": bug_ids}
+            )
+            if response.status_code == 200:
+                break
+            elif response.status_code == 202:
+                # All the results are not ready yet, try again in 1 second
+                time.sleep(retry_sleep)
+            else:
+                response.raise_for_status()
+        else:
+            total_sleep = retry_count * retry_sleep
+            msg = f"Couldn't get {len(bug_ids)} bug classification in {total_sleep} seconds, aborting"
+            raise Exception(msg)
+
+        json_response = response.json()
+
+        # Inject back the bug in the response
+        if bugs:  # Deprecated
```

Comment:
?

Reply:
It is still used by the others bugbug scripts https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/scripts/defectenhancementtask.py#L101